_sass/news.scss

```diff
@@ -91,10 +91,10 @@ section:nth-child(even) .newsitem {
     }
     .title {
       padding: 0 0 10px 30px;
-      background: no-repeat 10px 0 url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="10"><path d="M5,0l0,1000" stroke="%23ccc"/><circle r="5" cx="5" cy="15" fill="%23ccc"/></svg>');
```

Comment:
Dass das nicht base64 ist war gewollt, da es so kleiner ist. Geht das mit dem Plugin auch?

Reply:
Nein, so ein Format wird von dem Plugin nicht unterstützt; da fällt immer Base64 raus.
Wobei laut https://codepen.io/tigt/post/optimizing-svgs-in-data-uris das "rohe" SVG auch gar nicht wirklich in Data URLs vorkommen darf, sondern mindestens URL-encoded sein muss. Und in der Tat zeigt der IE11 im Moment bei https://ffhb.de/blog/ die SVGs nicht an.

Für komplett korrektes und minimales Encoding müsste man wohl sowas wie bei https://github.com/tigt/mini-svg-data-uri machen, also im SVG-Inhalt die nötigsten Zeichen URL-encoden.

Ich fänds OK, die SVGs auch als Base64 zu encoden und den zusätzlichen Platzverbrauch zu akzeptieren, wenn dafür die Wartbarkeit verbessert wird. Wer dann Zeit und Lust hat, kann ja das Plugin noch um einen SVG-Modus erweitern.

Und ja, die anderen PNGs waren in der Tat schon optimiert.